packages/string/src/case/utils/regex.ts

```
@@ -0,0 +1,115 @@
+export const MYANMAR_REGEX = /\p{Script=Myanmar}/u;
+export const ORIYA_REGEX = /\p{Script=Oriya}/u;
+export const SINHALA_REGEX = /\p{Script=Sinhala}/u;
+export const TAMIL_REGEX = /\p{Script=Tamil}/u;
+export const TELUGU_REGEX = /\p{Script=Telugu}/u;
+export const THAI_REGEX = /\p{Script=Thai}/u;
+export const TIBETAN_REGEX = /\p{Script=Tibetan}/u;
+// Special modifiers for Uzbek Latin script
+export const UZBEK_LATIN_MODIFIER_REGEX = /[\u02BB\u02BC\u0027]/u;
+
+// eslint-disable-next-line no-control-regex,regexp/no-control-character
+export const FAST_ANSI_REGEX = /(\u001B\[[0-9;]*[a-z])/i;
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)